app/schemas/general_schema.rb

```diff
@@ -44,7 +44,7 @@ class GeneralSchema < ActiveTriples::Schema
   property :copyright_jurisdiction, predicate: ::RDF::Vocab::PREMIS.hasCopyrightJurisdiction
 
   # xsd:anyURI with CVs
-  property :rights_override, predicate: VisibilityService::PREDICATE_RIGHTS_OVERRIDE, multiple: false
+  # property :rights_override, predicate: VisibilityService::PREDICATE_RIGHTS_OVERRIDE, multiple: false
```

Comment:
Yep, good question. I was thinking just about leaving it as a marker here for now and we can clean it up later with others predicates that were commented out earlier once we finalize all the predicates. But yes, we can do it now if you prefer.